walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,89 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
-}
+
+    /** Check if the event and the request contains the same attendee. */
+    private boolean containSameAttendees(Collection < String > eventAttendees, Collection < String > requestAttendees) {
```

Comment:
Hm, the name suggests you're checking if the request and the events contain all of the same attendees, but it looks like you're just checking if there's some attendee in common. Maybe name it something like "containsAttendeesInCommon"?